docs/releases/1.8.5.txt

```diff
@@ -29,3 +29,6 @@ Bugfixes
 
 * Alphabetized ordering of imports in ``from django.db import migrations,
   models`` statement in newly created migrations (:ticket:`25384`).
+
+* Fixed adding a text or a blob field with unhashable default to MySQL
```

Comment:
@scop Could you add the `unhashable` word to `docs/spelling_wordlist` so the docs spelling check does not fail on it? (please keep the list in alphabetical order).